custom_functions/add_evidence.py

```diff
@@ -0,0 +1,40 @@
+def add_evidence(container_id=None, object_id=None, content_type=None, **kwargs):
+    """
+    Custom function implementation of REST API for Evidence
+    
+    Args:
+        container_id (CEF type: phantom container id): Id of the container to which you are adding evidence.
+        object_id (CEF type: *): Id of object to be added - artifact id, note id, etc.
+        content_type (CEF type: *): The content type of the object to add as evidence. One of the types:
+            containerattachment
+            artifact
+            actionrun
+            container
+            note
+    
+    Returns a JSON-serializable object that implements the configured data paths:
+        id (CEF type: *): ID of the evidence item
+    """
+    ############################ Custom Code Goes Below This Line #################################
+    import json
+    import phantom.rules as phantom
+    
+    outputs = {}
+    # Ensure content_type is lowercase
+    content_type = content_type.lower()
+    data = {
+        "container_id": container_id,
+        "object_id": object_id,
+        "content_type": content_type,
+        }
+    # Build url for evidence endpoint
+    url = phantom.build_phantom_rest_url('evidence')
+    # Post data to evidence endpoint
+    response = phantom.requests.post(uri=url, json=data, verify=False).json()
+    # If successful add evidence id to outputs
+    if response.get('success') == 'true':
```

Comment:
First of all, I think this check is incorrect, because the success value should be True, and True does not equal 'true'.

Second, if you run this against an invalid content object or content type, there is currently no error. I think we should check if the response contains `'failed': True` as well. If it does not contain 'success' or `failed', then it is also no good.